plutus-ledger/plutus-ledger.cabal

```diff
@@ -1,158 +1,166 @@
-    default: False
-    manual: True
+  description:
+    Defer errors from the plugin, useful for things like Haddock that can't handle it.
+
+  default:     False
+  manual:      True
 
 library
-    import: lang
-    hs-source-dirs: src
-    default-language: Haskell2010
-    exposed-modules:
-        Data.Time.Units.Extra
-        Ledger
-        Ledger.Address
-        Ledger.AddressMap
-        Ledger.Blockchain
-        Ledger.CardanoWallet
-        Ledger.Crypto
-        Ledger.Fee
-        Ledger.Generators
-        Ledger.Index
-        Ledger.Index.Internal
-        Ledger.Orphans
-        Ledger.Params
-        Ledger.TimeSlot
-        Ledger.Tokens
-        Ledger.Tx
-        Ledger.Tx.CardanoAPI
-        Ledger.Typed.Tx
-        Ledger.Typed.Scripts
-        Ledger.Typed.Scripts.Validators
-        Ledger.Typed.TypeUtils
-        Ledger.Value
-        Ledger.Validation
-    reexported-modules:
-        Data.Aeson.Extras,
-        Data.Either.Extras,
-        Plutus.V1.Ledger.Ada as Ledger.Ada,
-        Plutus.V1.Ledger.Bytes as Ledger.Bytes,
-        Plutus.V1.Ledger.Credential as Ledger.Credential,
-        Plutus.V1.Ledger.DCert as Ledger.DCert,
-        Plutus.V1.Ledger.Interval as Ledger.Interval,
-        Plutus.V1.Ledger.Slot as Ledger.Slot,
-        Plutus.V1.Ledger.Scripts as Ledger.Scripts,
-        Plutus.V1.Ledger.TxId as Ledger.TxId,
-        Plutus.V1.Ledger.Time as Ledger.Time,
-        -- The rest of the plutus-ledger-api modules are reexported from within
-        -- the Haskell modules and not in the current cabal file.
-        -- For example: Plutus.V1.Ledger.Address is reexported by Ledger.Address
-    other-modules:
-        Ledger.Tx.CardanoAPITemp
+  import:             lang
+  hs-source-dirs:     src
+  default-language:   Haskell2010
+  exposed-modules:
+    Data.Time.Units.Extra
+    Ledger
+    Ledger.Address
+    Ledger.AddressMap
+    Ledger.Blockchain
+    Ledger.CardanoWallet
+    Ledger.Crypto
+    Ledger.Fee
+    Ledger.Generators
+    Ledger.Index
+    Ledger.Index.Internal
+    Ledger.Orphans
+    Ledger.Params
+    Ledger.TimeSlot
+    Ledger.Tokens
+    Ledger.Tx
+    Ledger.Tx.CardanoAPI
+    Ledger.Typed.Scripts
+    Ledger.Typed.Scripts.Validators
+    Ledger.Typed.Tx
+    Ledger.Typed.TypeUtils
+    Ledger.Validation
+    Ledger.Value
+
+  reexported-modules:
+    Data.Aeson.Extras,
+    Data.Either.Extras,
+    Plutus.V1.Ledger.Ada as Ledger.Ada,
+    Plutus.V1.Ledger.Bytes as Ledger.Bytes,
+    Plutus.V1.Ledger.Credential as Ledger.Credential,
+    Plutus.V1.Ledger.DCert as Ledger.DCert,
+    Plutus.V1.Ledger.Interval as Ledger.Interval,
+    Plutus.V1.Ledger.Slot as Ledger.Slot,
+    Plutus.V1.Ledger.Scripts as Ledger.Scripts,
+    Plutus.V1.Ledger.TxId as Ledger.TxId,
+    Plutus.V1.Ledger.Time as Ledger.Time,
 
-    build-depends:
-        base >=4.9 && <5,
-        aeson -any,
-        array -any,
-        bytestring -any,
-        cardano-api -any,
-        cardano-api:gen -any,
```

Comment:
The build is failing because `cardano-api:gen` is now missing. This seems like a bug in the formatter?